CHANGELOG.md

```diff
@@ -13,8 +13,12 @@ and **Merged pull requests**. Critical items to know are:
 
 The versions coincide with releases on pip. Only major versions will be released as tags on Github.
 
+## [0.1.17](https://github.com/oras-project/oras-py/tree/main) (0.1.17)
```

Comment:
Did you forget to remove this extra section?

Reply:
fixed.